### PR TITLE
Doc update

### PR DESCRIPTION
**Pull request by sojanpr**
Adding notes on new Android  Content indexing  updates
@aaustin @ethanneff @Sarkar

### CODE REVIEW

**Comment by sojanpr** [comment]
Please  do not  merge this until Android  v2.12.0 is released

**Comment by sojanpr** [comment]
Can we get this reviewed and merged. The  Android SDK release  is out now

File: pages/marketing-channels/google-app-indexing.md [hunk 1]
```diff
@@ -223,7 +225,7 @@ If you'd like to view all associated customizations with the Branch Universal Ob
 
 ## Enable App Indexing for Google on Branch
 
-If you have completed the prerequisites, you've done the hard part! Now you should go enable automatic sitemap generation on the [Organic Search](https://dashboard.branch.io/search) page of the Branch Dashboard. Check the `Automatic sitemap generation` checkbox.
+If you have completed the prerequisites, you've done the hard part! Now you should go enable automatic sitemap generation on the [Organic Search](https://dashboard.branch.io/search) page of the Branch Dashboard. Check the `Automatic sitemap generation` checkbox. Alternatively you can index your BranchUniversalObjects on Google search and local private content indesing using "BranchUniversalObject#listOnGoogleSeach" API.
```
Comment (aaustin) [inline pages/marketing-channels/google-app-indexing.md]: I'd change this addition to:  `Note that if you're using the "BranchUniversalObject#listOnGoogleSeach" API, your content will automatically appear in the local search index but not in public Google search for new users until you check this box.`

**Comment by aaustin** [comment]
@sojanpr we're not updating this repo any more. We've migrated over to https://github.com/BranchMetrics/docs

**Pull request closed without merging**